Author: Patrycja Rosa <alpine@ptrcnull.me>
Date:   Tue Jan 18 19:04:58 2022 +0100
Patch-Source: https://github.com/SarutaSan72/Yass/pull/95

    Remove reference to com.sun package

    (it's not even used anyway)

diff --git a/src/yass/I18.java b/src/yass/I18.java
--- a/src/yass/I18.java
+++ b/src/yass/I18.java
@@ -18,8 +18,6 @@

 package yass;

-import com.sun.xml.internal.ws.api.ResourceLoader;
-
 import javax.imageio.ImageIO;
 import javax.swing.*;
 import java.awt.*;
